examples/gridViewExample/src/main/java/io/realm/examples/realmgridview/GridViewExampleActivity.java

```diff
@@ -119,7 +119,8 @@ public boolean shouldSkipClass(Class<?> clazz) {
                 .create();
 
         JsonElement json = new JsonParser().parse(new InputStreamReader(stream));
-        List<City> cities = gson.fromJson(json, new TypeToken<List<City>>() {}.getType());
+        List<City> cities = gson.fromJson(json, new TypeToken<List<City>>() {
```

Comment:
Well, in the original version is pretty easy to miss the fact that it's an anonymous class there